.golangci.yml

```diff
@@ -0,0 +1,65 @@
+service:
+  golangci-lint-version: 1.23.6 # Keep this in sync with .circleci/config.yml and Jenkinsfile.pr
```

Comment:
1.23.6 is not yet in the golangci service. You can use 1.23.3. I have a PR open to extend it till 1.23.6. Also remove the part about Jenkinsfile.pr.